src/main/java/uk/gov/justice/digital/delius/service/ConvictionService.java

```diff
@@ -405,23 +404,19 @@ private LocalDate previouslyKnownTerminationDateOf(Offender offender) {
     }
 
     private Boolean inBreachOf(Offender offender) {
-        if (!probationStatusOf(offender).equals(ProbationStatus.CURRENT)) {
+        if (probationStatusOf(offender) != ProbationStatus.CURRENT) {
             return null;
         }
-        return Optional.of(offender)
-            .map(o ->
-                activeEvents(o.getEvents())
-                    .stream()
-                    .anyMatch(event -> event.getInBreach() == 1L)
-            ).orElse(null);
+        return offender.getActiveEvents()
+            .stream()
+            .anyMatch(event -> event.getInBreach() == 1L);
     }
 
     private Boolean preSentenceActivityOf(Offender offender) {
-        return Optional.of(offender)
-            .map((o) -> o.getEvents()
-                .stream()
-                .anyMatch(event -> event.getDisposal() == null)
-            ).orElse(false);
+        return offender.getActiveEvents()
```

Comment:
change in behaviour here - only look at active events to see if a disposal is missing

Reply:
@chris-faulkner-mojdt I think Peter is correct here, right now if there was offender who back in that day had a pre-sentence event raised which was then terminated without a sentence (e.g. they changed their plea and was found not guilty) the old code we treat that has "pre sentence" which is not correct since it is historic and there is no longer any pre-sentence active.